includes/REST/AdminOnboardingController.php

```diff
@@ -0,0 +1,385 @@
+<?php
+
+namespace WeDevs\Dokan\REST;
+
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_Error;
+
+/**
+ * Admin Onboarding REST Controller
+ *
+ * @since DOKAN_SINCE
+ */
+class AdminOnboardingController extends DokanBaseAdminController {
+    /**
+     * API namespace
+     *
+     * @var string
+     */
+    protected $namespace = 'dokan/v1';
+
+    /**
+     * API base
+     *
+     * @var string
+     */
+    protected $rest_base = 'admin-onboarding';
+
```

Comment:
Please use `DokanBaseAdminController`'s `namespace` and `rest_base` rules. Is there any resone to override namespace property?